cpp/tests/io/csv_test.cpp

```diff
@@ -1081,6 +1081,48 @@ TEST_F(CsvReaderTest, ArrowFileSource)
   expect_column_data_equal(std::vector<int8_t>{9, 8, 7, 6, 5, 4, 3, 2}, view.column(0));
 }
 
+TEST_F(CsvReaderTest, ZippedCSV)
```

Comment:
I think we could have this test in Python without embedding the compressed file. I'll give it a try today.

Reply:
Here's the equivalent Python test, AFAICT:
```Python
def test_small_zip(tmpdir):
    df = pd.DataFrame(
        {
            "a": [1997] * 2,
            "b": ["Ford"] * 2,
            "c": ["Super, luxurious truck"] * 2,
        }
    )

    fname = tmpdir.join("small_zip_file.zip")
    df.to_csv(fname, index=False)

    got = cudf.read_csv(fname)
    assert_eq(df, got)

```
Got a segfault locally with this test, hopefully it's the same segfault :)

Reply:
Thanks for preparing the python test! I've added it and removed the C++ in favour of the python test.